store/postgres/src/relational.rs

```diff
@@ -42,6 +44,23 @@ use crate::block_range::{BLOCK_RANGE_COLUMN, BLOCK_UNVERSIONED};
 pub use crate::catalog::Catalog;
 use crate::entities::STRING_PREFIX_SIZE;
 
+lazy_static! {
+    /// Experimental: a list of fully qualified table names that contain
+    /// entities that are like accounts in that they have a relatively small
+    /// number of entities, with a large number of change for each entity. It
+    /// is useful to treat such tables special in queries by changing the
+    /// clause that selects for a specific block range in a way that makes
+    /// the BRIN index on block_range usable
+    ///
+    /// Example: GRAPH_ACCOUNT_TABLES=sgd21902.pair,sgd1708.things
+    static ref ACCOUNT_TABLES: HashSet<String> = {
+        env::var("GRAPH_ACCOUNT_TABLES")
+            .ok()
+            .map(|v| v.split(",").map(|s| s.to_owned()).collect())
+            .unwrap_or(HashSet::new())
+    };
+}
```

Comment:
Is there a way to not duplicate this static variable?

Reply:
Yes, it's not needed in `block_range.rs`; I moved it from there to `relational.rs` and then forgot to delete it.